llvm/lib/Target/AMDGPU/AMDGPUISelLowering.cpp

```diff
@@ -29,6 +29,39 @@
 
 using namespace llvm;
 
+static bool CC_AMDGPU_Custom_I1(unsigned ValNo, MVT ValVT, MVT LocVT,
+                                CCValAssign::LocInfo LocInfo,
+                                ISD::ArgFlagsTy ArgFlags, CCState &State) {
+  static bool IsWave64 = static_cast<const GCNSubtarget &>(
+                             State.getMachineFunction().getSubtarget())
```

Comment:
getSubtarget<GCNSubtarget.() is cleaner than using your own static_cast

Reply:
Done.